challenges-api/src/main/java/org/exoplatform/challenges/model/RestEntity.java

```diff
@@ -0,0 +1,36 @@
+package org.exoplatform.challenges.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.exoplatform.social.core.space.model.Space;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class RestEntity implements Cloneable {
```

Comment:
For consistency, since you will add other Rest entities
```suggestion
public class ChallengeRestEntity implements Cloneable {
```

Reply:
Done !!